src/test/kotlin/blackjack/model/StateTest.kt

```diff
@@ -0,0 +1,53 @@
+package blackjack.model
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+private fun Hand(vararg cards: Card): Hand {
+    return Hand(cards.toList())
+}
+
+private fun Card(value: Int): Card {
+    return Card(CardNumber.entries.find { it.value == value }!!, CardShape.HEART)
+}
+
+class StateTest {
+    @Test
+    fun `카드의 숫자합이 기준점을 넘지 않으면 초기 카드 상태는 Hit이다`() {
+        val threshold = 21
+        val state = State.setupState(Hand(Card(10), Card(9)), threshold)
+        assertThat(state is Hit).isTrue()
```

Comment:
```suggestion
        assertThat(state).isInstanceOf(Hit::class.java)
```

테스트를 할 때도 is 연산자 대신, assertj 문법을 더 적극적으로 활용해주시면 좋을 것 같습니다.